assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -88,38 +91,46 @@ function DatePicker({ currentDate, onChange, onViewChange }) {
   );
 
   useEffect(() => {
-    updateTabIndexes();
+    // Wait one tick for the calendar to be available.
+    const timeout = setTimeout(() => {
+      updateTabIndexes();
+    });
+    return () => {
+      clearTimeout(timeout);
+    };
   }, [updateTabIndexes]);
 
   return (
-    <CalendarWrapper ref={nodeRef}>
-      <Calendar
-        value={value}
-        onChange={handleOnChange}
-        onViewChange={onViewChange}
-        onActiveStartDateChange={() => updateTabIndexes(true /* Set focus */)}
-        nextAriaLabel={_x(
-          'Next',
-          'This label can apply to next month, year and/or decade',
-          'web-stories'
-        )}
-        prevAriaLabel={_x(
-          'Previous',
-          'This label can apply to previous month, year and/or decade',
-          'web-stories'
-        )}
-        next2AriaLabel={_x(
-          'Jump forward',
-          'This label can apply to month, year and/or decade',
-          'web-stories'
-        )}
-        prev2AriaLabel={_x(
-          'Jump backwards',
-          'This label can apply to month, year and/or decade',
-          'web-stories'
-        )}
-      />
-    </CalendarWrapper>
+    <Suspense fallback={null}>
```

Comment:
Here I also think a properly sized fallback would make the swap a bit cleaner. Perhaps even with a spinner? (again note the throttling)

![slow-calendar](https://user-images.githubusercontent.com/637548/125837866-37d29c77-8917-496a-804f-7ffbf3648dcb.gif)

Reply:
Added:
![Screenshot 2021-07-16 at 11 37 25](https://user-images.githubusercontent.com/3294597/125918825-494ade7e-a6a7-4962-a249-a0c519e1d1bd.png)